CMakeLists.txt

```diff
@@ -278,10 +295,8 @@ IF (BUILD_SHARED_LIBRARY)
     TARGET_INCLUDE_DIRECTORIES (${SHARED_LIBRARY_TARGET} PUBLIC "${PROJECT_SOURCE_DIR}/include/")
     TARGET_INCLUDE_DIRECTORIES (${SHARED_LIBRARY_TARGET} PUBLIC "${PROJECT_SOURCE_DIR}/src/")
 
-    TARGET_LINK_LIBRARIES (${SHARED_LIBRARY_TARGET} "stdc++")
     TARGET_LINK_LIBRARIES (${SHARED_LIBRARY_TARGET} "pthread")
-    TARGET_LINK_LIBRARIES (${SHARED_LIBRARY_TARGET} "Boost::filesystem" "Boost::log" ${Boost_STACKTRACE_BASIC_LIBRARY})
-    TARGET_LINK_LIBRARIES (${SHARED_LIBRARY_TARGET} "dl")
```

Comment:
This was needed for stacktrace basic, surprised it's not needed for backtrace.

Reply:
Confirmed it works even with a fresh docker environment 👍. All good

Reply:
Awesome thanks for checking.